### PR TITLE
fix(sidepanel): use buttons

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

the current sidepanel use `<a href="#"/>`

**What is the new behavior?**

the new sidepanel use `<button class="btn btn-link" role="button">`

**Does this PR introduce a breaking change?**

- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

the styles may be impacted
**Other information**:

### CODE REVIEW

**Comment by jsomsanith** [comment]
The hover behavior is different from when we had `<a>`.
The paddings are not aligned with the guidelines anymore.

File: src/SidePanel/SidePanel.component.js [hunk 1]
```diff
@@ -45,15 +45,16 @@ function SidePanel(props) {
 			<ul className={listClassName}>
 				{actions.map(action => (
 					<li key={action.label}>
-						<a
-							href="#"
+						<Button
```
Comment (jsomsanith) [inline src/SidePanel/SidePanel.component.js]: The hover behavior is different from when we had <a>.
The paddings are not aligned with the guidelines anymore.

Comment (frassinier) [inline src/SidePanel/SidePanel.component.js]: We should treat this issue into [Talend/bootstrap-theme](https://github.com/Talend/bootstrap-theme), and this PR should not be blocked?

Comment (jsomsanith) [inline src/SidePanel/SidePanel.component.js]: :+1:

**Comment by frassinier** [comment]
I will fix unit tests

**Pull request merged**